src/Bicep.Core.Samples/Files/InvalidResources_CRLF/Completions/autoScalerPropertiesRequireEscaping.json

```diff
@@ -32,7 +32,7 @@
     "detail": "expander",
     "documentation": {
       "kind": "markdown",
-      "value": "Type: `'least-waste' | 'most-pods' | 'random'`  \n"
+      "value": "Type: `'least-waste' | 'most-pods' | 'random'`  \n  \n"
```

Comment:
I assume those are without any description, but are the additional empty lines necessary?

Reply:
Good catch. Looks like this is coming through from type generation - we legitimately have some properties with description `""`: https://github.com/Azure/bicep-types-az/blob/9db80f44909805c0498e3fb0137fb47f307959f3/generated/containerservice/microsoft.containerservice/2020-09-01/types.md#schemas30_autoscalerprofile

I'll create an issue in the type generation repo to nullify empty/whitespace strings as I don't think it should be Bicep's responsibility to normalize these during type loading.